src/components/Authentication/Login.jsx

```diff
@@ -0,0 +1,149 @@
+                  type="email"
+                  onChange={({ target }) => setEmail(target.value)}
+                  placeholder="Email"
+                  borderColor={"#CBD5E0"}
+                  borderRadius= '3px'
+                />
+                <Input
+                  style={{ width: '360px', height: '48px', margin: '20px' }}
+                  type="password"
+                  onChange={({ target }) => setPassword(target.value)}
+                  placeholder="Password"
+                  borderColor={"#CBD5E0"}
+                  borderRadius= '3px'
+                />
+              </Box>
+
+              <Box
+                style={{
+                  marginTop: '25px',
+                  marginBottom: '25px',
+                }}
+              >
+                <Link href='/signup'>
+                  <Button
+                    style={{
+                      borderRadius: '30px',
+                      borderColor: '#155696',
+                      borderWidth: '1.5px',
+                      marginRight: '16px',
+                      paddingLeft: '80px',
+                      paddingRight: '80px',
+                      width: '140px',
+                      height: '38px',
+                    }}
+                    backgroundColor={'#FFFFFF'}
+                    color={'#155696'}
+                    variant='outline'
+                    onMouseOver={(e) => {
+                      e.target.style.backgroundColor = '#E0E0E0'; 
+                    }}
+                    onMouseOut={(e) => {
+                      e.target.style.backgroundColor = '#FFFFFF';
+                    }}
+                  >
+                    Create Account
+                  </Button>
+                </Link>
+                <Button
+                  type="submit"
+                  style={{
+                    borderRadius: '30px',
+                    marginLeft: '16px',
+                    paddingLeft: '80px',
+                    paddingRight: '80px',
+                    width: '140px',
+                    height: '38px',
+                  }}
+                  backgroundColor={'#243268'}
+                  color={'#ffffff'}
+                  onMouseOver={(e) => {
+                    e.target.style.backgroundColor = '#1A2559'; 
+                  }}
+                  onMouseOut={(e) => {
+                    e.target.style.backgroundColor = '#243268';
+                  }}
+                >
+                  Login
+                </Button>
+              </Box>
+
+              <Box>
+                <a href="/forgotpassword" style={{ textDecoration: "underline"}}>
+                  Forgot Password
+                </a>
+              </Box>
+            </FormControl>
+          </form>
+        </Box>
+      </Center>
+    </Box>
+  );
+};
+
+Login.propTypes = {
+  cookies: instanceOf(Cookies).isRequired,
+};
+
+export default withCookies(Login);
```

Comment:
⚠️ **[eslint]** <react-refresh/only-export-components> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Fast refresh can't handle anonymous components. Add a name to your export.